coordinator-work.md

```diff
@@ -0,0 +1,61 @@
+
+## Uncrawlable Spreadsheet structure
+
+- While eventually we hope to be able to develop an integrated web-based application that will facilitate the entire workflow, for now we have to rely on a system based on Google Spreadsheets to manage harvesting activities. 
+
+- Uncrawlable Action spreadsheets 
+  - An Uncrawlable Action spreadsheets contains no more than 500 URLs
+  - It is divided into tabs (one per role) 
+    - with formulas that help populate rows from one tab to the other
+  - It contain a number of column per tab to support various aspects of the role's work
+  - Each DataRescue event is attributed one separate Action spreadsheet
+    - The event participants can continue working on it remotely after the event until it is completed
+    - Or alternatively they could "give it back" to the organizers at the end of the event
+
+- An Uncrawlable Index spreadsheet 
+  - A single Uncrawlable Index spreadsheet is used to keep track of all URLs being managed in the project
+  - Using formulas, it automatically compiles a list of all URLS listed in all Uncrawlable Action spreadsheets.
+  - This spreadsheet only contains minimal information about each URL (URL, UUID, Done/Not Done)
+  - It is not meant to be edited in any way. 
+  - However its can be downloaded and imported into a tool like OpenRefine to help with deduplication.
+  
+- Seeders spreadsheet
+  - The Chrome Extension used by the Seeders automatically populate a separate spreadsheet. 
+  - We have found that keeping that spreadsheet separate helps with the workflow
+
+## How each Uncrawlable Action spreadsheet is populated
+- It will include URLs coming from two main sources:
+   - URLs that were nominated by Seeders at a previous DataRescue event
+   - URLs that were identified througth the Union of Concerned Scientists survey, which asked the scientific community to list the most vulnerable and important data currently on accessible through federal websites.
+ 
+## Spreadsheet Minder
+- A Spreadsheet Minder (or Spreadsheet Minder Team) is in charge with managing the Uncrawlable Index and Action spreadsheets
+- Here are the task they are responsible for:
+  - Prepare new Uncrawlable Action spreadsheets by moving URLs from the Seeders spreadsheet and the Survey into new Uncrawlable Action spreadsheets.
+  - Generate UUIDs and add them to the Uncrawlable Action spreadsheets, making sure that each URL has a UUID
+  - Add the name of the event (in the "Event Name" cell)
+  - Add a rough importance rating (in the "Importance" cell) 
+    - E.g., a URL coming from the Survey would automatically get a high importance rating
+  - Keep an eye on all spreadsheets making sure that everything is in order.
+  - Regularly take snapshots of each spreadsheet in case an problem occurs and the content of a spreadsheet needs to get recovered
+  - Use the Index spreadsheet to track overall progress (ratio Done / Not Done) and to help with deduplication efforts
+ 
+## UUID generation
```

Comment:
I'm not sure if this belongs in here? This seems like a smaller point, not a huge coordination component